packages/compass-import-export/src/components/export-modal.tsx

```diff
@@ -94,12 +95,18 @@ function ExportOptions({
     [ns, query]
   );
 
+  // In the LeafyGreen package `@leafygreen-ui/radio-group` version `10.0.3` the
+  // radio component doesn't listen to the `darkMode` that the `LeafyGreenProvider` provides.
+  // So for new we are setting it ourselves here.
+  const darkMode = useDarkMode();
```

Comment:
Asked about this in the slack channel: https://mongodb.slack.com/archives/CFFA5BS79/p1672946315602849
If there's a ticket made I'll add a TODO linking it.

Reply:
https://jira.mongodb.org/browse/LG-2741